src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -21,7 +21,9 @@ import { reportError, withErrorReporting } from 'src/libs/error'
 import { versionTag } from 'src/libs/logos'
 import * as Nav from 'src/libs/nav'
 import { notify } from 'src/libs/notifications'
-import { appIsSettingUp, convertedAppStatus, currentApp, currentAttachedDataDisk, getGalaxyCost, isCurrentGalaxyDiskDetaching } from 'src/libs/runtime-utils'
+import {
```

Comment:
Nit: My preference would be to either put these on the line with the curly braces, as they were before, or newline each of them/put a newline in the middle, so we are making better use of the improved horizontal space.

Reply:
Sorry, I ran Intellij's "Organize Imports" script and it did this. I'll revert it.